src/client/task.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"context"
+	"io"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/grpcutil"
+	"github.com/pachyderm/pachyderm/v2/src/taskapi"
+)
+
+// ListTask lists tasks in the given namespace and group
+func (c APIClient) ListTask(namespace string) ([]*taskapi.TaskInfo, error) {
+	ctx, cancel := context.WithCancel(c.Ctx())
+	defer cancel()
+
+	stream, err := c.TaskClient.ListTask(
+		ctx,
+		&taskapi.ListTaskRequest{
+			Namespace: namespace,
+		},
+	)
+	var out []*taskapi.TaskInfo
+	if err != nil {
+		return nil, grpcutil.ScrubGRPC(err)
+	}
+	for {
+		ti, err := stream.Recv()
+		if errors.Is(err, io.EOF) {
```

Comment:
Would you change this, to this:
```
if err != nil {
	if errors.Is(err, io.EOF) {
		break
	}
	return err
}
```
It reads a bit better imo, and it is a pattern I have been gradually moving our client wrappers to.